selectors-4/Overview.bs

```diff
@@ -4259,7 +4259,7 @@ while still preserving as much of the <em>usefulness</em> of '':visited'' as pos
 	containing [=tuples=] of three pieces of information:
```

Comment:
On line 4254 above, I wanted to note that we recently had an exploit reported which takes advantage of Chrome filtering out 4XX-5XX status codes from History to leak the existence of subpages via :visited history (for more information please see https://github.com/explainers-by-googlers/Partitioning-visited-links-history/issues/8). 

This is an issue we are unable to resolve with partitioning but are working to resolve via a separate solution. However, we won't be able to get rid of the mitigations entirely until this issue is resolved.